test/Modules/ModulesGraded.jl

```diff
@@ -1176,3 +1176,15 @@ end
   @test Int(degree(gen(N, 1))[2]) == -2
 end
 
+@testset "all_monomials for graded free modules" begin
+  R, (x, y, u, v, w) = QQ[:x, :y, :u, :v, :w]
+  S, (x, y, u, v, w) = grade(R)
+
+  F = graded_free_module(S, [-1, 2])
+
+  for d in -4:4
+    amm = Oscar.all_monomials(F, d)
+    @test d < -1 || !isempty(amm)
+    @test all(x->degree(x) == grading_group(F)([d]), amm)
+  end
+end
```

Comment:
Why is this identical test in ModulesGraded.jl and in GradedModules.jl?
(By the way, is the latter filename the result of a mishap or intentional?)

Reply:
I put the tests in one file and ran it locally. Tests failed completely, but not due to the part I added. I suppose that test-file was deprecated, but not removed. Just to be sure, I then put my tests in both, finally. Somebody should tell us what's up with the test files, though.